packages/contracts-bedrock/src/L1/ProtocolVersions.sol

```diff
@@ -5,9 +5,7 @@ import { OwnableUpgradeable } from "@openzeppelin/contracts-upgradeable/access/O
 import { ISemver } from "src/universal/interfaces/ISemver.sol";
 import { Storage } from "src/libraries/Storage.sol";
 import { Constants } from "src/libraries/Constants.sol";
-
-/// @notice ProtocolVersion is a numeric identifier of the protocol version.
-type ProtocolVersion is uint256;
```

Comment:
Removing this so that it is only declared once (in the interface). This
prevents confusing build errors.

Reply:
If we switch to using `IProtocolVersions` everywhere, like in `DeploySuperchain`, would that resolve the build error? Checked with @smartcontracts and ideally we don't need to import a contract's own interface in a contract. This lets us run `forge inspect` on a contract to generate it's interface automatically, but if a contract imports it's interface we can't do that since there'd be a circular dependency

Reply:
Looks like @AmadiMichael opened a PR that changes the new scripts to use interfaces. Should we try merging that one first then rebase this on top, and it may remove the need for this? https://github.com/ethereum-optimism/optimism/pull/12217

Reply:
Yeah, that SGTM. It looks like there is very little conflict between the two PRs, so I'm just going resolve the other comment then move on to building on top of this one while waiting for #12217 to go in.